recipe/meta.yaml

```diff
@@ -11,11 +11,11 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   {{ hash_type }}: {{ hash_val }}
-  # patches:
-  #   - python35-compat.patch  # [py35]
+  patches:
+     - python35-compat.patch  # [py35]
```

Comment:
I actually stole this patch from upstream - it's mentioned on some issue mailing lists; see [here](https://bitbucket.org/catherinedevlin/cmd2/issues/18/python-35-renames-subprocessmswindows)